### PR TITLE
Add basic structure and content to readme file

### PR DESCRIPTION
**Pull request by mmhernandez**
Added basic structure and content to the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,46 @@
...
+  4. Invite friends or family over and tell some jokes!
+
 
-Here is Gillian's Readme comment for testing:
-Hello team!
+## Tech Used / Dependencies
+* Vanilla JavaScript
+* Add-on packages:  
+    * ...
+* Color inspiration from: ...
```
Comment (GillianTrethewey) [inline README.md:39]: Here is the code for the images for your convenience:

```html
    <h3 align="left">Languages and Tools:</h3>
    <p align="left">
      <img
        src="https://raw.githubusercontent.com/devicons/devicon/master/icons/html5/html5-original-wordmark.svg"
        alt="html5"
        width="40"
        height="40"
      />
      <img
        src="https://raw.githubusercontent.com/devicons/devicon/master/icons/css3/css3-original-wordmark.svg"
        alt="css3"
        width="40"
        height="40"
      />
      <img
        src="https://raw.githubusercontent.com/devicons/devicon/master/icons/javascript/javascript-original.svg"
        alt="javascript"
        width="40"
        height="40"
      />
      <img
        src="https://www.vectorlogo.zone/logos/git-scm/git-scm-icon.svg"
        alt="git"
        width="40"
        height="40"
      />
    </p>
```

**Review by GillianTrethewey — CHANGES_REQUESTED** [review]
Looks like a good start! I've added the code in the markup for your convenience to insert it where you would like to - it will show the images for each tool we used. :)

File: README.md [hunk 2]
```diff
@@ -1,14 +1,69 @@
+
+
+(Note - include the icons for the stack + team members)
```
Comment (GillianTrethewey) [inline README.md:3]: Please remove this note since you've already included the icons and team members.

Comment (mmhernandez) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (mmhernandez) [reply README.md:56]: Added

**Comment by mmhernandez** [comment]
The requested changes have been implemented and pushed

File: README.md [hunk 3]
```diff
@@ -1,14 +1,63 @@
...
+    height="40"
+    />
+</p>
+
+
+
+## Contributors
+The Team |  GitHub  | LinkedIn
+:---:  |  :---:  |  :---:
+Lyle Trethewey  |  [![git hub logo](https://github.com/chingu-voyages/v42-toucans-team-02/images/github.png "Lyle's github")](https://github.com/LyleT)  |   [![linkedin logo](https://github.com/chingu-voyages/v42-toucans-team-02/images/linkedin.png "Lyle's LinkedIn")](https://www.linkedin.com/in/lyle-trethewey/)
```
Comment (LyleT) [inline README.md:61]: Hi Melissa - these images are not rendering. Are you able to fix this problem? You can view the readme by clicking the three dots under the review changes button. 8)

Comment (mmhernandez) [reply README.md:61]: @LyleT - will they only display once the merge is complete? The URLs are pointing to where we are housing the images inside the repo.

**Review by LyleT — CHANGES_REQUESTED** [review]
Please fix GitHub and Linkedin images... 8)

**Comment by mmhernandez** [comment]
For now, I will remove the images.

File: README.md [hunk 4]
```diff
@@ -1,14 +1,62 @@
...
+    alt="git"
+    width="40"
+    height="40"
+    />
+</p>
+
+
+
+## Contributors
+Lyle Trethewey
```
Comment (GillianTrethewey) [inline README.md:59]: Melissa, thanks in advance for putting back the code for all the links? That will make it easier the next time to add the images if you don't get time today. If you do get time, here is the code again for the image links:
```html
<img src="https://skillicons.dev/icons?i=github" />
<img src="https://skillicons.dev/icons?i=linkedin" />

USAGE:
<a href="https://github.com/gilliantrethewey"><img src="https://skillicons.dev/icons?i=github" /></a>
```

Appreciated!

**Review by GillianTrethewey — CHANGES_REQUESTED** [review]
Please see the comment about the links and added the html for your convenience to add the images. Thanks!!

**Comment by mmhernandez** [comment]
Readme updated

**Review by GillianTrethewey — APPROVED** [review]
Looks excellent Melissa!

**Review by LyleT — APPROVED** [review]

**Pull request merged**